### PR TITLE
Adding android app banners

### LINKED ISSUE #6692: Android Banner on Mobile Site
Now that we have the Android app up and running, when Android users are browsing the mobile site on their phones, we should add a banner that they can click to download the Android app.

### PR DESCRIPTION
**Pull request by kelsin**
Fixes #6692

I'm not sure if we need to add this to all three manifest sections or not, but for now it is. This uses https://github.com/kudago/smart-app-banner via bower, but we could move to another plugin if people have a preference. I used some images that were already in the project for this, but if there are better ones we can update it :)

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -41,7 +41,8 @@
     "ngInfiniteScroll": "1.0.0",
     "pnotify": "1.3.1",
     "sticky": "*",
-    "swagger-ui": "wordnik/swagger-ui#v2.0.24"
+    "swagger-ui": "wordnik/swagger-ui#v2.0.24",
+    "smart-app-banner": "*"
```
Comment (paglias) [inline bower.json]: @kelsin could you specify a specific version for the package? We don't want a future update to break the site

Comment (kelsin) [inline bower.json]: Right now the github repo for the plugin has no tags so bower can't reference the package via a version number. We can install the package via npm but I don't think that works with your build system at the moment?

Added a comment to an open issue for the plugin: https://github.com/kudago/smart-app-banner/issues/37

Comment (paglias) [inline bower.json]: I see... what about selecting a specific commit?

**Comment by kelsin** [comment]
Using a specific commit of smart app banner now.

**Comment by paglias** [comment]
@kelsin thanks! one last thing, can you post a screenshot of how this would look?

**Comment by kelsin** [comment]
Unfortunately afk all day today, but will get this up tomorrow :)

**Comment by kelsin** [comment]
Android App Banner

**Comment by crookedneighbor** [comment]
Hmmm... why is it the chest icon instead of the griffin?

**Comment by kelsin** [comment]
I'm not an artist so I just used the image that is named apple-touch-icon for this plugin. If there is another image I should use just point me to it and I'll update the commit.

**Comment by Alys** [comment]
@kelsin There's a few other icons in the same location as the one you used. Can you look through them and if one of them is the purple Gryphon icon, please use that?

**Comment by kelsin** [comment]
The only one I see is favicon. I will update to it tonight and post a screenshot.

**Comment by kelsin** [comment]
The favicon_192x192 file had transparent background so it didn't look good. Took one of the files in the marketing folder and croped it to get a version of the white gryphon on purple background.

**Comment by kelsin** [comment]
I had it at 192x192 which depending on the browser resize algorithm would look bad. Now it's set to 57x57 and looks like https://kelsin.smugmug.com/Blog/i-zm9dSQQ/0/O/habitica-logo-smart-banner.png

Github is caching the image, so using the link is the best way to see the current look.

**Comment by Alys** [comment]
@lemoness Is this what you're hoping for?

screenshot

**Comment by lemoness** [comment]
Yes! That looks good. Can the button say "GET" instead?

**Comment by kelsin** [comment]
Yes it can, will edit and update screenshots soon.

**Comment by crookedneighbor** [comment]
@kelsin What's the state of this?

**Comment by kelsin** [comment]
@crookedneighbor sorry work got crazy, but I need to update the text on the buttons to "GET" instead of "VIEW". Doing that now.

**Comment by kelsin** [comment]
@lemoness buttons now say "GET" instead of "VIEW"

**Comment by kelsin** [comment]
Any word on the review of this? Would love to get it merged to get my contributor badge and maybe move onto slightly bigger tasks!

**Comment by crookedneighbor** [comment]
Sorry for the wait @kelsin 

One question I have. I already get a banner on the site on my iOS device. Will this conflict with the one that is already there?

**Comment by kelsin** [comment]
I have not done extensive device testing (due to not having anything by my single phone) but according to the plugin documentation it shouldn't.

The plugin only targets iOS devices that are too old to get the native banners: https://github.com/kudago/smart-app-banner/blob/master/index.js#L62

**Comment by crookedneighbor** [comment]
This will count towards a contributor tier! What's your User ID? You can find it in [Settings > API](https://habitrpg.com/#/options/settings/api). Be sure to only copy the User ID and _not_ your API Token.

**Comment by kelsin** [comment]
User ID: 39f233ca-07fb-4a7e-b7bd-a49afd6aa71b

**Comment by crookedneighbor** [comment]
Tier awarded!

**Comment by crookedneighbor** [comment]
You can read more about [contributor tiers and rewards here](http://habitrpg.wikia.com/wiki/Contributor_Rewards).

Note that as your tier increases, it takes progressively more work to achieve the next one, so for the higher tiers, larger PRs or a few PRs are required. There's some information about that at Contributor Rewards, but feel free to ask here if you have questions!

**Comment by kelsin** [comment]
No questions! The rewards make sense and I started on a small easy ticket
for a reason :) Thanks!

**Pull request merged**